docs/usermanual.md

```diff
@@ -9,7 +9,7 @@
 ## API Overview
 
 This section describes the specific South-bound API implemented by this IoTAgent. For the Configuration API and other
-APIs concerning general IoTAgents, check the [API Reference section](#apireference);
+APIs concerning general IoTAgents, check the [API Reference section](https://github.com/Keshav-NEC/iotagent-ul/blob/master/README.md#install).
```

Comment:
Sorry, but this is not acceptable.

You cannot include links to your owns repos in this one.